src/events/models/Event.ts

```diff
@@ -197,3 +197,8 @@ export interface INewEvent {
   slug: string;
   title: string;
 }
+
+export interface IEventColumn {
```

Comment:
I would say it makes more sense to have this in the actual component file, because it is not a *model* for how events look like, but rather what `IProps` the component takes in, also the `event_type` should be of the type `EventTypeEnum`, not a number

Reply:
^agree